spinalcordtoolbox/deepseg/models.py

```diff
@@ -477,6 +485,18 @@ def is_up_to_date(path_model):
     return True
 
 
+def get_citation_info(name_model):
+    """
+    Get citation information for a given model.
+
+    :param name_model: str: Name of model.
+    :return: str: Citation information.
+    """
+    if name_model not in MODELS:
+        raise ValueError(f"Model '{name_model}' not found in MODELS")
+    return MODELS[name_model].get('citation', None)
```

Comment:
I don't think we need a whole new function for this? The code is simple enough that we can just put it directly inside `sct_deepseg`.

Also, since `sct_deepseg` already contains the conditional `if name_model in list(models.MODELS.keys()):`, we probably don't need the `ValueError` check since we can safely assume that the model will be present.